authors/apps/articles/models.py

```diff
@@ -18,3 +18,7 @@ class Article(models.Model):
     is_deleted = models.BooleanField(default=False)
     is_published = models.BooleanField(default=False)
     author = models.ForeignKey(User, on_delete=models.CASCADE)
+    user_id_likes = models.ManyToManyField(
+        User, related_name='likes', blank=True)
+    user_id_dislikes = models.ManyToManyField(
+        User, related_name='dislike', blank=True)
```

Comment:
Would you like to use `dislikes` since `likes` is plural?

Reply:
Yeah let me change that